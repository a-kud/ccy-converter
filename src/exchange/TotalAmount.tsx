import { getCurrencySign } from "../utils/currency";
import { CurrencyAmountType } from "../types";

export function TotalAmount({ currency, amount }: CurrencyAmountType) {
  return (
    <div>
      You have {getCurrencySign(currency)}
      {amount}
    </div>
  );
}
