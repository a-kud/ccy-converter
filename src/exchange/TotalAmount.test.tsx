import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { TotalAmount } from "./TotalAmount";

describe("TotalAmount", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<TotalAmount amount="24.00" currency="USD" />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
