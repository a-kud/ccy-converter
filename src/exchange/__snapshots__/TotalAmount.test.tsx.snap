// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TotalAmount renders correctly 1`] = `
<div>
  You have 
  $
  24.00
</div>
`;
